services/spar/src/Spar/API.hs

```diff
@@ -606,14 +628,14 @@ idpCreate ::
     Member IdPRawMetadataStore r,
     Member (Error SparError) r
   ) =>
-  Maybe UserId ->
+  (UserId, TeamId) ->
```

Comment:
nitpick: can you curry it? or is it costly?

Reply:
Technically it can be, but it would move the ugliness into the API type: since this is the handler for an API endpoint then having two separate arguments for it would mean that the API type would also consist of two parts like so: `AuthProtect "UserId" :> AuthProtect "TeamId" :> ...` (while currently it is `AuthProtect "zoptuser"`).